src/index.html

```diff
@@ -17,11 +17,365 @@
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star"></div>
+              <p class="stars__reviews">Reviews: 5</p>
+            </div>
+          </div>
+          <div class="card__price">
+            <span class="card__price-label">Price:</span>
+            <span class="card__price-value">$2,199</span>
+          </div>
+          <button
+            class="card__button"
+            data-qa="hover"
+          >
+            BUY
+          </button>
+        </div>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__photo"></div>
+        <div class="card__info">
+          <p class="card__title">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="card__code">Product code: 195434</p>
+          <div class="stars">
+            <div class="stars__container">
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star"></div>
+              <p class="stars__reviews">Reviews: 5</p>
+            </div>
+          </div>
+          <div class="card__price">
+            <span class="card__price-label">Price:</span>
+            <span class="card__price-value">$2,199</span>
+          </div>
+          <button
+            class="card__button"
+            data-qa="hover"
+          >
+            BUY
+          </button>
+        </div>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__photo"></div>
+        <div class="card__info">
+          <p class="card__title">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="card__code">Product code: 195434</p>
+          <div class="stars">
+            <div class="stars__container">
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star"></div>
+              <p class="stars__reviews">Reviews: 5</p>
+            </div>
+          </div>
+          <div class="card__price">
+            <span class="card__price-label">Price:</span>
+            <span class="card__price-value">$2,199</span>
+          </div>
+          <button
+            class="card__button"
+            data-qa="hover"
+          >
+            BUY
+          </button>
+        </div>
+      </div>
```

Comment:
The product details in the card elements are repeated multiple times. If this is for testing purposes, it's fine, but for a real application, you should ensure that each card displays unique product information.